monitoring/alerts.yaml

```diff
@@ -132,3 +135,17 @@ groups:
     annotations:
       description: "Latency of delete object operations is more than 1s"
       summary: "Very high delete latency"
+
+  # As a platform admin I want to be alerted (warning) when the utilization metrics service is enabled
+  # but not available during the last 5 minutes
+  - alert: UtilizationMetricsServiceNotAvailable
+    expr: |
+      sum(rate(s3_cloudserver_utilization_service_state{namespace="${namespace}",state="healthy"}[1m])) * 100
+          / sum(rate(s3_cloudserver_utilization_service_state{namespace="${namespace}",state=~"^(unhealthy|healthy)$"}[1m]))
```

Comment:
```suggestion
          / sum(rate(s3_cloudserver_utilization_service_state{namespace="${namespace}",state=~"unhealthy|healthy"}[1m]))
```

Reply:
Done in https://github.com/scality/cloudserver/pull/5556/commits/95ef7fcd9db20f1ae3b11e5590c27e7a76fc45cf